Sources/TarkaUI/Components/Input Field/Model/TUIInputFieldItem.swift

```diff
@@ -20,6 +20,9 @@ public struct TUIInputFieldItem: Equatable, Hashable {
   public var style: TUIInputFieldStyle
   public var title: String = ""
   public var value: String = ""
+  public var highlightBarColor: Color = .primaryTUI
+  public var isHighlightBarEnabled: Bool = false
+  public var stateType: TUIInputFieldState = .none
```

Comment:
Better rename to `inputFieldStateType` for better readability

Reply:
> Better rename to `inputFieldStateType` for better readability

As this is already in the TUIInputFieldItem, just stateType is good enough. If you can see the title, value they are not named as inputFieldStyle or InputFieldTitle. Kindly consider this!

Reply:
As this is already in the TUIInputFieldItem, just stateType is good enough as like title, value etc.